src/main/java/com/epam/rd/autotasks/springemployeecatalog/constants/DepartmentEnum.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.rd.autotasks.springemployeecatalog.constants;
+
+public enum DepartmentEnum {
+    managerDepartmentId("ID_d_m"),
+    employeeDepartmentId("ID_d_e"),
+    managerDepartmentName("NAME_d_m"),
+    employeeDepartmentName("NAME_d_e"),
+    managerDepartmentLocation("LOCATION_d_m"),
+    employeeDepartmentLocation("LOCATION_d_e");
```

Comment:
It would be better to place here deparment column names. And to add _d_e during extraction.